src/client.rs

```diff
@@ -28,6 +28,24 @@ static DEFAULT_USER_AGENT: &'static str = concat!(env!("CARGO_PKG_NAME"), "/", e
 ///
 /// The `Client` holds a connection pool internally, so it is advised that
 /// you create one and reuse it.
+///
+/// # Examples
+///
+/// ```no_run
+/// # extern crate reqwest;
```

Comment:
FYI, you shouldn't need this line in doc examples. rustdoc will automatically include the crate being tested and all dependencies.